src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,57 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] NUMBERS = new int[12];
+    private static final int MAX_VALUE = 11;
+    private static final int DEFAULT_INDEX = -1;
+    private static final int CAPACITY = 12;
 
-    public int total = -1;
+    private final int[] NUMBERS = new int[getCAPACITY()];
 
-    public void countIn(int in) {
+    public static int getMaxValue() {
+        return MAX_VALUE;
+    }
+
+    public static int getDefaultIndex() {
+        return DEFAULT_INDEX;
+    }
+
+    public static int getCAPACITY() {
+        return CAPACITY;
+    }
+
+    public int getTotal() {
```

Comment:
Shouldn't be encapsulated (typical private field). This is an exaggeration. Using private fields internally usually do not require encapsulation.